packages/nuxt-locokit/src/layouts/WithAsideNav.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div class="relative flex h-screen">
+    <aside
+      tabindex="-1"
+      class="static flex flex-shrink-0 overflow-hidden bg-white border-r focus:outline-none"
+    >
+      <!-- Mini navigation -->
+      <div
+        class="bg-primary flex flex-col flex-shrink-0 h-full px-2 py-4 border-r"
+      >
+        <!-- Brand -->
+        <div class="flex-shrink-0">
+          <NuxtLink class="h-16 inline-block" :to="{ name: ROUTES_NAMES.HOME }">
+            <span class="sr-only">{{ runtimeConfig.public.PROJECT_NAME }}</span>
+            <img alt="logo" class="h-16" src="/assets/logo-mobile-alt.svg" />
```

Comment:
I think this should be a prop of the layout.
When we want to customize the instance for a customer, this should be set somewhere...